app/src/main/java/com/ataulm/stacks/StackFragment.kt

```diff
@@ -0,0 +1,29 @@
+package com.ataulm.stacks
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import kotlinx.android.synthetic.main.fragment_stack.*
+
+class StackFragment : Fragment() {
+
+    private val args: StackFragmentArgs by navArgs()
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+        return inflater.inflate(R.layout.fragment_stack, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        helloAndroid.text = args.stackId
+        openStackButton.setOnClickListener {
+            val action = StackFragmentDirections.actionStackFragmentSelf("child stack")
```

Comment:
"child stack" is just some text to display while I was checking how this works. 

When this is done correctly, we'll pass the ID of the stack that was clicked.